python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
If this is bad enough to push out a second release so quickly can we at least add an issue to find out how/why this fairly innocuous line is causing infinite loops.
I'm happy to see a 6.1.16 without it but I think this warrants a closer look after this